src/main/resources/config/application-dev.yml

```diff
@@ -112,3 +112,23 @@ jhipster:
             queue-size: 512
     audit-events:
         retention-period: 120 # Number of days before audit events are deleted.
+
+# Properties to be exposed on the /info management endpoint
+info:
+    guided-tour:
+        courseShortName: 'artemistutorial'
+        tours:
+            - key: 'cancel_tour'
```

Comment:
would it not make more sense to use 'cancel_tour' as key and the '' as value?

Reply:
Yes, I changed the structure of the configuration